dasgoclient-binary.spec

```diff
@@ -1,11 +1,11 @@
-### RPM cms dasgoclient-binary v02.04.38
+### RPM cms dasgoclient-binary v02.04.39
 Source0: git+https://github.com/dmwm/dasgoclient?obj=master/%{realversion}&export=dasgoclient&output=/dasgoclient.tar.gz
 Source1: git+https://github.com/dmwm/cmsauth?obj=master/e9fca92e3335252a5f71d8e6d09c64012f7d3c0c&export=github.com/dmwm/cmsauth&output=/cmsauth.tar.gz
 Source2: git+https://github.com/vkuznet/x509proxy?obj=master/c93f6cae85114060b3b65861ea8436e4e14c54a6&export=github.com/vkuznet/x509proxy&output=/x509proxy.tar.gz
 Source3: git+https://github.com/buger/jsonparser?obj=master/6bd16707875b997f7a60327f888a28a3d28cf8c2&export=github.com/buger/jsonparser&output=/jsonparser.tar.gz
 Source4: git+https://github.com/go-mgo/mgo?obj=v2/3f83fa5005286a7fe593b055f0d7771a7dce4655&export=gopkg.in/mgo.v2&output=/mgo.v2.tar.gz
 Source5: git+https://github.com/pkg/profile?obj=master/3a8809bd8a80f8ecfe4ee1b34b3f37194968617c&export=github.com/pkg/profile&output=/profile.tar.gz
-Source6: git+https://github.com/dmwm/das2go?obj=master/a2bf102e509e1251fce4f6d09c6e68a486c49dae&export=github.com/dmwm/das2go&output=/das2go.tar.gz
+Source6: git+https://github.com/dmwm/das2go?obj=master/cba4d3ebdd1407dc978b63ab15f1cb220f936432&export=github.com/dmwm/das2go&output=/das2go.tar.gz
```

Comment:
@vkuznet , are you still using this spec to buiild dasgoclient?

Reply:
yes, the valid hash tag for das2go library should be cba4d3ebdd1407dc978b63ab15f1cb220f936432

Reply:
previously you were using the spec to build/ipload the rpm which we ere using to extract the `dasgoclient` binaries but now I see that we get these binaries directly from the github. so whatis the use of this spec?

Reply:
I only keep it for backward compatibility, but it is not required since binaries are shipped from github directly. It is on my TODO list to clean-up the spec but I think it should be done independently from the release patch(es), i.e. this PR.

Reply:
we already have cleaned up cmsdist and removed the unused specs. `dasgoclient-binary.spec` was already removed from `IB/CMSSW_12_3_X/master` branch that is why this PR can not be merged. I can open a new PR with dasgoclient.spec change only